witness/src/Dscp/Witness/Launcher/Mode.hs

```diff
@@ -79,6 +80,11 @@ type WitnessWorkMode ctx m =
 newtype TxRelayInput = TxRelayInput { getTxRelayInput :: STM.TQueue GTxWitnessed }
 newtype TxRelayPipe  = TxRelayPipe  { getTxRelayPipe  :: STM.TQueue GTxWitnessed }
 
+relayTx :: WitnessWorkMode ctx m => GTxWitnessed -> m ()
```

Comment:
Can you move it elsewhere? Defining newtypes in `Mode` is more or less fine, but defining logic is not. At least it doesn't feel right somehow.

Reply:
I'll make `Relay` module.